src/schema/v2/auction_result.ts

```diff
@@ -141,6 +141,35 @@ const AuctionResultType = new GraphQLObjectType<any, ResolverContext>({
         }
       },
     },
+    performance: {
+      type: new GraphQLObjectType({
+        name: "AuctionLotPerformance",
+        fields: {
+          mid: {
+            type: GraphQLString,
+            description: "Percentage performance over mid-estimate",
+            resolve: ({
+              hammer_price_cents,
+              high_estimate_cents,
+              low_estimate_cents,
+            }) => {
+              if (
+                hammer_price_cents &&
```

Comment:
based on conversation in slack think we want to update this to use `realized_price_cents` rather than `hammer_price_cents` 
https://artsy.slack.com/archives/C01B2P6LJUU/p1611936317214600